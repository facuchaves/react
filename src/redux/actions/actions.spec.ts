import {searchEntities} from '.';

test('searchEntities action', () => {
  expect(searchEntities({q: 'query'})).toMatchSnapshot();
});
