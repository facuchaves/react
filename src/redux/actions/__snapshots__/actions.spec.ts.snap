// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchEntities action 1`] = `
Object {
  "payload": Object {
    "query": Object {
      "q": "query",
    },
  },
  "type": "SEARCH_ENTITIES",
}
`;
